backend/constants.js

```diff
@@ -0,0 +1,18 @@
+import User from './models/user';
+
+export const SAMPLE_USER = User({
+  firstName: 'Test',
+  lastName: 'User',
+  password: 'password',
+  phone: 'number',
+  location: 'location',
+  _id: '621bda9026575a70d82448c1',
+});
+
+export const SAMPLE_USER2 = User({
+  firstName: 'Test2',
+  lastName: 'User2',
+  password: 'password2',
+  phone: 'number2',
+  location: 'location2'
+});
```

Comment:
I saw this minus sign next to this line. Is this our lint detection? Was just curious

Reply:
No, we don't have a linter yet. Might have manually deleted the empty line